README.md

```diff
@@ -16,22 +16,22 @@ This image comes in many flavours, based on different Ubuntu releases and packag
 
 ## Package versions
 
-| Package   | bionic        | focal            |
-| --------- | ------------- | -----------------|
-| Erlang    | 21.3 and 22.2 | 25.0.3           |
-| Elixir    | 1.9.4         | 1.13.0           |
-| Node.js   | -             | 14.16.1          |
-| Rust      | -             | 1.63.0           |
-| FFmpeg    | 4.2.2         | 4.2.2            |
-| SDL2      | ?             | ?                |
-| FDK AAC   | 2.0           | 2.0              |
-| Portaudio | 19.6.0        | 19.6.0           |
-| MAD       | 0.15.1        | 0.15.1           |
-| FLAC      | 1.3.2         | 1.3.2            |
-| Opus      | 1.1.2         | 1.1.2            |
-| libsrtp2  | -             | 2.3.0  	       |
-| libnice   | -             | master (833c1aa) |
-| libmp3lame| -             | 3.100            |
+| Package   | bionic        | focal            | jammy            |
+| --------- | ------------- | -----------------|------------------|
+| Erlang    | 21.3 and 22.2 | 25.0.3           | 25.0.3           |
+| Elixir    | 1.9.4         | 1.13.0           | 1.13.0           |
+| Node.js   | -             | 14.16.1          | 14.16.1          |
+| Rust      | -             | 1.63.0           | 1.63.0           |
+| FFmpeg    | 4.2.2         | 4.2.2            | 4.4.2            |
+| SDL2      | ?             | 2.0.10           | 2.0.20           |
+| FDK AAC   | 2.0.0         | 2.0.0            | 2.0.0            |
+| Portaudio | 19.6.0        | 19.6.0           | 19.6.0           |
+| MAD       | 0.15.1        | 0.15.1           | 0.15.1           |
+| FLAC      | 1.3.2         | 1.3.2            | 1.3.3            |
+| Opus      | 1.1.2         | 1.1.2            | 1.3.1            |
+| libsrtp2  | -             | 2.3.0  	       | 2.4.2            |
+| libnice   | -             | master (833c1aa) | master (833c1aa) |
```

Comment:
it's no longer used, can be removed

Reply:
What is no longer used? The whole `##Package versions` section? Why? I mean I don't mind removing it, but as it is there it seems like it could be helpful... Or does the `###Default versions` sections is enough? (BTW - naming isn't best there I guess.

Reply:
Nope, just libnice :P that's the line the comment is attached to ;)

Reply:
<img width="942" alt="Zrzut ekranu 2022-08-29 o 17 49 19" src="https://user-images.githubusercontent.com/11754737/187241789-069532d1-0fcc-4d60-9af2-2f1eebafbe0d.png">


Then GH wasn't showing it precisely - I saw 4 lines, and couldn't imagine Opus isn't used...

Reply:
Yup, GitHub always shows a few lines before the one to which the comment is attached. It's quite misleading

Reply:
and thus subject is better then pronoun in such case: `libnice is no longer used` vs `it's no longer used` :)